bedrock/firefox/views.py

```diff
@@ -666,6 +668,20 @@ def get_template_names(self):
         return [template]
 
 
+class WhatsNewIndiaView(WhatsnewView):
+    def get_template_names(self):
+        template = super(WhatsNewIndiaView, self).get_template_names()
```

Comment:
No need to pass anything to `super()` anymore now that we're on Python 3.7.

```suggestion
        template = super().get_template_names()
```

There is also no need to do this at the beginning of this method. You can call the above on line 682 so that it only does that work if it needs to.